docs/src/pages/components/index.md

```diff
@@ -10,24 +10,19 @@ hideTitle: true
 unProse: true
 prependJs:
   - "import Note from '../../../../src/components/note'"
-  - "import OverviewHeader from '../../../../src/components/overview-header'"
   - "import App from '../../app.js'"
-  - "const version = require('../../../../package.json').version; // eslint-disable-line"
+overviewHeader:
+  title: 'Dr. UI'
+  features:
+    - 'React components to build documentation sites'
+    - 'Support for IE 11 and all modern browsers'
+  changelogLink: /dr-ui/changelog/
+  installLink: https://github.com/mapbox/dr-ui/blob/main/README.md
+  ghLink: https://github.com/mapbox/dr-ui
+  # version: "" dynamic version is set in pages-shell.js
```

Comment:
We'll definitely want to be able to set this dynamically (using `constants.json`) including for multi-structured sites. That should be possible in the `PageShell` in each site, right? Just wanted to flag that in case that's a complexity you hadn't considered yet!

Reply:
> That should be possible in the PageShell in each site, right?

Exactly! I tested this and it will work for multi-structure, too.